nat/src/stateful/mod.rs

```diff
@@ -188,27 +188,25 @@ impl StatefulNat {
         state: NatFlowState<I>,
         idle_timeout: Duration,
     ) {
-        fn session_timeout_time(timeout: Duration) -> Instant {
-            Instant::now() + timeout
-        }
```

Comment:
What's the matter with keeping this in a dedicated function? We've talked of replacing `Instant::now()`, so it seems better to me to hide it from the rest of the function.